docs/UserGuide.md

```diff
@@ -114,14 +114,33 @@ contains two types of formatting.
 | `<condition>`    | Contents enclosed between "<>" are the inputs needed for the command to be valid. <br /> Not all commands needs input.         |
 | `/commandAction` | Content after "/" is to classify the action of the command. <br/> Such as classifying it to be list / delete / update commands |
 
-<span class="box warning">⚠️**Heads Up!** Your inputs cannot contain the pipe character `|`!</span>
+<span class="box warning">:warning: **Heads Up!** Your inputs cannot contain the pipe character `|`!</span>
 
 ### Finding Your Way Around The Application
 * [Workout Commands](#workout-commands)
 * [Plan Commands](#plan-commands)
 * [Schedule Commands](#schedule-commands)
 * [Search Commands](#search-commands)
 
+<span class="box warning">:warning: **Heads Up!** Due to the design of WerkIt!, commands, `/commandAction`, and
+`<condition>` must be separated by a single whitespace. Excessive whitespaces will not be accepted by the application.
+However, plan names may have more than a single whitespace in between words.<br/><br/>
+**Examples**<br/><br/>
+<span class="box>
+:x: `workout /new    push up   /reps   1000`<br/><br/>
+:white_check_mark: `workout /new push up /reps 1000`
+</span>
+<span class="box">
+:x: `plan     /list`</span>
+:white_check_mark: `plan /list`</span>
+</span>
+<span class="box">
+:x: `plan /new Leg Day   /workouts   1, 2, 3`<br/<br/>
+:white_check_mark: `plan /new Leg Day /workouts 1, 2, 3`<br/><br/>
+:white_check_mark: `plan /new Leg     Day /workouts 1, 2, 3`
+</span>
+</span>
+
```

Comment:
Not sure how the github pages looks like but here is the current view of the md file in github at my side:
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/69451253/161442814-05592e59-ff87-489e-a9e6-9a5a1352f5f5.png)

Reply:
PR is still in drafting stage 😬